ngraph/test/runtime/interpreter/opset_int_tbl.hpp

```diff
@@ -50,6 +50,10 @@ NGRAPH_OP(Elu, op::v0)
 NGRAPH_OP(Selu, op::v0)
 NGRAPH_OP(Ceiling, op::v0)
 NGRAPH_OP(Gelu, op::v0)
-
-
+NGRAPH_OP(BatchNormInference, op::v0)
+NGRAPH_OP(ReverseSequence, op::v0)
+NGRAPH_OP(Convert, op::v0)
 NGRAPH_OP(CTCLoss, op::v4)
+NGRAPH_OP(OneHot, op::v1)
```

Comment:
Just as a request for the future, let's keep this tbl file divided to sections according to opset version and do not mix up everything together